### DISCUSSION TITLE
Tasmota devs working on Matter support (open-source firmware for Espressif ESP32 SoC series)

### DISCUSSION THREAD

**Discussion by Hedda** [mfucci/node-matter#224] [Matter compatible devices / controllers]
FYI, Tasmota developer [s-hadinger](https://github.com/s-hadinger) have now started development on Matter support for Tasmota (firmware for Espressif's ESP32 SoC series).

Follow here to track status and progress ->  https://github.com/arendst/Tasmota/discussions/17872

(Quote: _Only commissioning for now, but that's a big piece of protocol._ - [s-hadinger](https://github.com/s-hadinger) ).

(For reference, previous Tasmota disscussion -> https://github.com/arendst/Tasmota/discussions/13020#discussioncomment-4866679)

**Comment by Apollon77** [comment]
Thankyou, very interesting.
